my_store/blog/dashboard/apps.py

```diff
@@ -0,0 +1,26 @@
+# from django.apps import AppConfig
```

Comment:
Please remove unuse line, and order how to import correctly, See how each module should order by this document https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/